{
  "parameters": {
    "subscriptionId": "management.azure.com",
    "resourceGroupName": "db4e2fdb-6d80-4e6e-b7cd-736098270664",
    "deviceName": "Microsoft.DataBoxEdge",
    "url": "https://management.azure.com",
    "api-version": "2023-01-01-preview",
    "Accept": "application/json, text/json",
    "Authorization": "Sanitized",
    "User-Agent": "azsdk-net-ResourceManager.DataBoxEdge/1.1.0-alpha.20230329.1 (.NET 6.0.15; Microsoft Windows 10.0.22621)",
    "x-ms-client-request-id": "604a5582c25af7a370558a068fd73fd0",
    "x-ms-return-client-request-id": "true"
  },
  "responses": {
    "200": {
      "body": {
        "value": [
          {
            "properties": {
              "title": "Lost heartbeat from your device.",
              "alertType": "DeviceHeartBeat",
              "appearedAtDateTime": "2023-03-17T11:46:31.9379967Z",
              "recommendation": "If your device is offline, then the device is not able to communicate with the Azure service. This could be due to one of the following reasons: <br/> &nbsp;1. The Internet connectivity is broken. Check your internet connection. In the local web UI of the device, go to Troubleshooting > Diagnostic tests. Run the diagnostic tests. Resolve the reported issues.<br/> &nbsp;2. The device is turned off or paused on the hypervisor. Turn on your device! For more information, go to <a href=\"https://aka.ms/dbe-device-local-mgmt\" target=\"_blank\">Turn on your device</a>. <br/>&nbsp;3. Your device could have rebooted due to an update. Wait a few minutes and try to reconnect.",
              "severity": "Critical",
              "errorDetails": {
                "errorCode": "",
                "errorMessage": "",
                "occurrences": 1
              },
              "detailedInformation": {}
            },
            "id": "/subscriptions/db4e2fdb-6d80-4e6e-b7cd-736098270664/resourceGroups/demo-resources/providers/Microsoft.DataBoxEdge/dataBoxEdgeDevices/demo-edge-sdk6-2023/alerts/b117940e-4e3d-4d45-b83c-27234eb2d6cd",
            "name": "b117940e-4e3d-4d45-b83c-27234eb2d6cd",
            "type": "Microsoft.DataBoxEdge/dataBoxEdgeDevices/alerts"
          },
          {
            "properties": {
              "title": "Cluster resource 'demouser' is down. You device will continue to run IO, but you cannot use the local web UI to manage the device.",
              "alertType": "HcsClusterResource.ReplicatedLocalUserFaulted",
              "appearedAtDateTime": "2019-06-13T20:56:32.8546114Z",
              "recommendation": "If the issue persists, collect a log package and contact Microsoft Support. For more information, go to <a href=\"https://aka.ms/collect-logs\">Use the local web UI to collect a log package</a>.",
              "severity": "Warning",
              "errorDetails": {
                "errorCode": "",
                "errorMessage": "",
                "occurrences": 1
              },
              "detailedInformation": {}
            },
            "id": "/subscriptions/db4e2fdb-6d80-4e6e-b7cd-736098270664/resourceGroups/demo-resources/providers/Microsoft.DataBoxEdge/dataBoxEdgeDevices/demo-edge-sdk6-2023/alerts/389258e9-5547-4018-97c1-24ef20ac1a1c",
            "name": "389258e9-5547-4018-97c1-24ef20ac1a1c",
            "type": "Microsoft.DataBoxEdge/dataBoxEdgeDevices/alerts"
          }
        ]
      }
    }
  }
}